### ISSUE TITLE
Bug when creation bootstrap key with [SPACE] in name 

### ISSUE THREAD

**Issue by av-sebastian** [exoscale/terraform-provider-exoscale#183]
Creation a ssh_key with a [SPACE] in the name gives an error, but it created the key correctly in exoscale.
On of the problem with this is, that the terraform state is not getting written correctly.

```
[sb--> ] terraform-main $ tf -chdir=environments/exoscale/01-public-keys/ apply

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # module.bootstrap-key.exoscale_ssh_key.this will be created
  + resource "exoscale_ssh_key" "this" {
      + fingerprint = (known after apply)
      + id          = (known after apply)
      + name        = "Bootstrap only"
      + public_key  = "{OMITTED}"
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

module.bootstrap-key.exoscale_ssh_key.this: Creating...
╷
│ Error: Get "https://api-ch-gva-2.exoscale.com/v2/ssh-key/Bootstrap%20only": invalid request: Invalid request signature
│ 
│   with module.bootstrap-key.exoscale_ssh_key.this,
│   on ../../../modules/exoscale/ssh-key/main.tf line 1, in resource "exoscale_ssh_key" "this":
│    1: resource "exoscale_ssh_key" "this" {
│ 
╵
```

**Comment by PhilippeChepy** [comment]
Hello @av-sebastian,

Thank you for the bug report. This should be resolved in the next release since https://github.com/exoscale/terraform-provider-exoscale/pull/184 is addressing the signature issue. The next release will be available very soon.

**Comment by kobajagi** [comment]
Fixed in v0.38.0
